docs/hugo/content/introduction/annotations.md

```diff
@@ -0,0 +1,27 @@
+# Annotations understood by the operator
+
+## Annotations specified by the user
+
+Note that unless otherwise specified, allowed values are _case sensitive_ and should be provided in lower case.
+
+1. `serviceoperator.azure.com/reconcile-policy`: Specifies the reconcile policy to use. Allowed values are:
+    - `run`: The operator performs all actions as normal. This is the default if no annotation is specified.
```

Comment:
I think this is technically not needed. Generally I understand that one might want to be explicit, but in this case that's what anyone using the operator would expect anyway.

Reply:
I like the idea of enabling users to be explicit _even if_ it's the default behaviour.

Reply:
Agree this is technically not needed, but given the name of the property `reconcile-policy` it felt like we should have a name for the "normal" policy, even if people practically speaking aren't going to bother setting it explicitly.